src/main/java/com/yelp/nrtsearch/server/luceneserver/field/VectorFieldDef.java

```diff
@@ -143,11 +207,11 @@ protected VectorFieldDef(String name, Field requestField) {
     this.vectorDimensions = requestField.getVectorDimensions();
     this.elementType = requestField.getVectorElementType();
     if (isSearchable()) {
+      VectorSearchType vectorSearchType =
+          getSearchType(requestField.getVectorIndexingOptions(), elementType);
       this.similarityFunction = getSimilarityFunction(requestField.getVectorSimilarity());
       this.vectorsFormat =
-          requestField.hasVectorIndexingOptions()
-              ? createVectorsFormat(requestField.getVectorIndexingOptions())
-              : null;
+          createVectorsFormat(vectorSearchType, requestField.getVectorIndexingOptions());
```

Comment:
Do we want to add the vector format even when indexing options are not specified because we'll pick defaults? Is there a case where we would add a vector field but wouldn't want to add this?

Reply:
I think we will always want to create the vector format now, since otherwise it won't get the new size limit.

Reply:
This is in an `isSearchable()` block. The vector format is always null if search is false.

Reply:
Ah I missed that, I was looking for `getSearch()`. Thanks for clarifying!